user-guide/modules/ROOT/pages/release-process.adoc

```diff
@@ -1,69 +1,33 @@
-
-== Milestones in the Release Cycle
-
-There is a strict countdown to a public release.
-
-7 Weeks Prior to Release Date::
-
-The *master* branch is closed to all check ins, except bug fixes and quality checks.
-
-6 Weeks Prior to Release Date::
-
-The *master* branch is closed to major code changes. There can be no rewrites of code, even to fix issues.
+=== Regression Testing
 
-5 Weeks Prior to Release Date::
+Regression testing is an essential part of the release process, ensuring the quality and compatibility of the libraries. The Boost community maintains a set of regression tests, which are run on a diverse range of platforms and compilers. The tests are performed by volunteers who contribute their computing resources to the project.
 
-The *master* branch is closed to all check ins, except with permission from the release committee.
+The results of the regression tests are published on the Boost website, providing library maintainers and users with up-to-date information about the library's compatibility and performance. Library maintainers use this information to identify and fix any issues before a release.
 
-4 days Prior to Beta Release Date::
+== Beta and Final Release
 
-The *master* branch is closed. Beta release candidates are built.
+There is a strict countdown to a public release. Four weeks prior to the release date the Beta release is published to the Boost site.
 
-4 Weeks Prior to Release Date::
-
-The Beta release is published to the Boost site. The *master* branch is opened to small bug fixes and documentation changes. Permission from the release committee is required for larger changes.
-
-1 Week Prior to Release Date::
-
-The *master* branch is closed to all check ins, except high-priority fixes.
-
-4 Days Prior to Release Date::
-
-The *master* branch is closed. Release candidates are built.
-
-Day of Release::
-
-The release candidate is published to the Boost site. The *master* branch is opened for all check ins.
-
-+
-If issues are found with a release candidate that are important enough to address quickly (that is, before the next full public release), then a point release will be built when fixes are available and tested. This will not typically result in the *master* branch being closed to other check ins.
+If issues are found with a release that has gone public, and the issues are important enough to address quickly (that is, before the next full public release), then a point release will be built when fixes are available and tested. 
 
 == See Also
 
-* https://docs.cppalliance.org/user-guide/index.html[User Guide]
-* https://docs.cppalliance.org/contributor-guide/index.html[Contributor Guide]
-* https://docs.cppalliance.org/formal-reviews/index.html[Formal Reviews]
+For details of the Release Process that are pertinent to contributors, refer to the https://docs.cppalliance.org/contributor-guide/index.html[Contributor Guide].
```

Comment:
This link is wrong. It's pointing to a fixed version. If you click on it in the preview version, it will take you to the master branch docs, which has very old content.

It should be an asciidoc xref to the relevant section of the contributor guide.

Reply:
@anarthal Done, the actual link required is:
link:../contributor-guide/release-process.html[Contributor Guide Release Process]

Reply:
> link:../contributor-guide/release-process.html[Contributor Guide Release Process]

I don't think that's it. This link can also change.

You need an xref with more fields because it's another component.

See https://docs.antora.org/antora/latest/page/xref/